### PR TITLE
Remove amp-ad-loading-ux flag.

### LINKED ISSUE #6009: Clean up `amp-ad-loading-ux` experiment

### PR DESCRIPTION
**Pull request by lannka**
closes #6009

### CODE REVIEW

File: build-system/global-configs/prod-config.json [hunk 1]
```diff
@@ -14,7 +14,6 @@
   "ad-type-custom": 1,
   "amp-scrollable-carousel": 1,
   "amp-inabox": 1,
-  "amp-ad-loading-ux": 1,
```
Comment (zhouyx) [inline build-system/global-configs/prod-config.json]: we need to keep this until the change reach prod

Comment (lannka) [inline build-system/global-configs/prod-config.json]: done

File: extensions/amp-ad/0.1/test/test-amp-ad-ui.js [hunk 2]
```diff
@@ -35,8 +33,6 @@ describes.realWin('amp-ad-ui handler', {
     uiHandler = new AmpAdUIHandler(adImpl);
     uiHandler.setDisplayState(AdDisplayState.LOADING);
     // Always set to true since this is in PROD.
```
Comment (zhouyx) [inline extensions/amp-ad/0.1/test/test-amp-ad-ui.js]: nit: you can delete this line as well

File: src/layout.js [hunk 3]
```diff
@@ -277,11 +273,5 @@ export function getNaturalDimensions(element) {
  */
 export function isLoadingAllowed(element) {
   const tagName = element.tagName.toUpperCase();
-  if (tagName == 'AMP-AD' || tagName == 'AMP-EMBED') {
```
Comment (zhouyx) [inline src/layout.js]: need to keep this check. As it only applies to amp-ad.

Comment (lannka) [inline src/layout.js]: good catch

File: extensions/amp-ad/0.1/test/test-amp-ad-ui.js (hunk 2 above)
Comment (lannka) [inline extensions/amp-ad/0.1/test/test-amp-ad-ui.js]: done

**Comment by zhouyx** [comment]
Test fails due to https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js#L239
We insert the fallback placeholder. iframe is no longer the firstChild of the amp-ad element.

**Pull request merged**